src/pseudopeople/configuration/generator.py

```diff
@@ -75,12 +75,15 @@ def get_configuration(user_configuration: Union[Path, str, Dict] = None) -> Conf
     :param user_configuration: A path to the YAML file or a dictionary defining user overrides for the defaults
     :return: a ConfigTree object of the noising configuration
     """
-
-    if type(user_configuration) == str and user_configuration.lower() == Keys.NO_NOISE:
-        config_type = Keys.NO_NOISE
+    if isinstance(user_configuration, str) and user_configuration.lower() == NO_NOISE:
```

Comment:
You don't need to check the type, just if user_conriguration == NO_NOISE

Reply:
I think you forgot to simplify this `if user_configuratioin == NO_NOISE:`

Reply:
Agree with Steve, and you don't need to do `user_configuration.lower()` either